### PR TITLE
Fix imprecisely named arguments and variables

### PR DESCRIPTION
**Pull request by DoctorJohn**

### CODE REVIEW

File: cfmtoolbox/plugins/random_sampling.py [hunk 1]
```diff
@@ -11,12 +11,12 @@
 
 
 @app.command()
-def random_sampling(model: CFM, amount: int = 1) -> CFM:
+def random_sampling(model: CFM, number: int = 1) -> CFM:
```
Comment (DanielZhangDP) [inline cfmtoolbox/plugins/random_sampling.py:14]: this sounds weird?, is it really easy to understand from this name that it is the parameter for how many samples you get?

Comment (DoctorJohn) [reply cfmtoolbox/plugins/random_sampling.py:14]: What about something like `--num-samples`?

Comment (DanielZhangDP) [reply cfmtoolbox/plugins/random_sampling.py:14]: oki :)

File: docs/plugins/random-sampling.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+It generates a custom number of random valid configurations and outputs them into the console.
 
 The Random Sampling plugin requires the model to be bound which means no infinite upper bounds as instance cardinalities are allowed.
 In case of an unbound model, you can use other plugins like the Big M plugin to replace infinte upper bounds with finite ones.
 
 ## Usage
 
 Import a cfm and generate 5 random samples for it:
-The `--amount` parameter defaults to `1` if not specified.
+The `--number` parameter defaults to `1` if not specified.
```
Comment (DanielZhangDP) [inline docs/plugins/random-sampling.md:10]: this should be changed as well (as well as down below)

Comment (DoctorJohn) [reply docs/plugins/random-sampling.md]: oops :O

**Review by DanielZhangDP — APPROVED** [review]

**Pull request merged**